src/biotite/structure/transform.py

```diff
@@ -234,6 +234,78 @@ def rotate_about_axis(atoms, axis, angle, support=None):
     return _put_back(atoms, positions)
 
 
+def orient_principal_components(atoms):
```

Comment:
Would it be possible in your implementation to give an optional `order` parameter, e.g. `[2, 0, 1]` to align the 1st principal axis to *z*, the 2nd to *x* and the 3rd to *y*?